pages/Source_Code_Analysis_Tools.md

```diff
@@ -4,47 +4,63 @@ layout: col-sidebar
 title: Source Code Analysis Tools
 author: 
 contributors: 
-  - Dave Wichers, itamarlavender, will-obrien, Eitan Worcel, Prabhu Subramanian, kingthorin, coadaflorin, hblankenship, GovorovViva64, pfhorman, GouveaHeitor, Clint Gibler, DSotnikov, Ajin Abraham, Noam Rathaus   
+  - Dave Wichers, itamarlavender, will-obrien, Eitan Worcel, Prabhu Subramanian, kingthorin, coadaflorin, hblankenship, GovorovViva64, pfhorman, GouveaHeitor, Clint Gibler, DSotnikov, Ajin Abraham, Noam Rathaus, Mike Jang
 tags: source code analysis, static code analysis, tools
 permalink: /Source_Code_Analysis_Tools
 
 ---
 
 {% include writers.html %}
 
-[Source code analysis](Static_Code_Analysis) tools, also referred to as Static Application Security Testing (SAST) Tools, are designed to analyze source code or compiled versions of code to help find security flaws.
+[Source code analysis](Static_Code_Analysis) tools, also known as Static Application Security Testing (SAST) Tools, can help you analyze source code or compiled versions of code to help find security flaws.
 
-Some tools are starting to move into the IDE. For the types of problems that can be detected during the software development phase itself, this is a powerful phase within the development life cycle to employ such tools, as it provides immediate feedback to the developer on issues they might be introducing into the code during code development itself. This immediate feedback is very useful, especially when compared to finding
```

Comment:
I think OWASP does not need to discuss details of the Software Development Life Cycle.